cli/src/deploy/mod.rs

```diff
@@ -909,12 +909,14 @@ pub async fn upgrade(
 }
 
 async fn get_apps(status: bool) -> Vec<App> {
+  let config = get_config().await;
```

Comment:
`get_config()` will prompt the user to configure credentials and deploy configs if they are not properly setup so only start the spinner for the `/info` endpoint once we actually fire it